pytorch_lightning/plugins/training_type/deepspeed.py

```diff
@@ -276,6 +278,10 @@ def __init__(
         super().__init__(
             parallel_devices=parallel_devices, num_nodes=num_nodes, cluster_environment=cluster_environment
         )
+
+        self.checkpoint_plugin = DeepSpeedPlugin() if checkpoint_plugin is None else checkpoint_plugin
```

Comment:
```suggestion
        self.checkpoint_plugin = DeepSpeedCheckpointPlugin() if checkpoint_plugin is None else checkpoint_plugin
```
typo here?

Reply:
hahaha definitely, thanks for the catch!